.vib/sonarqube/goss/vars.yaml

```diff
@@ -0,0 +1,7 @@
+containerPorts:
+  http: 9090
+  elastic: 9091
```

Comment:
You should modify these default values.

Reply:
These are not default values. Those are 9000 and 9001